### PR TITLE
Update node version

### PR DESCRIPTION
**Pull request by Fllxd**

### CODE REVIEW

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
use the boundary values technique here

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -1,10 +1,53 @@
...
+    expect(getCoinCombination).toBeInstanceOf(Function);
+  });
+
+  it('should return [1, 0, 0, 0] for 1 cent', () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it('should return [1, 1, 0, 0] for 6 cents', () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+  });
```
Comment (djtyrf312) [inline src/getCoinCombination.test.js:16]:
```suggestion
```
you already have this test below

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -1,10 +1,53 @@
...
+
+  it('should return [0, 1, 0, 0]for 5 cents (boundary for 5-cent coin)', () => {
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+  });
+
+  it('should return [1, 1, 0, 0] for 6 cents (boundary after 5 cents)', () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+  });
+
+  it('should return[0, 0, 1, 0]for 10 cents(boundary for 10-cent coin)', () => {
```
Comment (djtyrf312) [inline src/getCoinCombination.test.js:42]: what about 9 coins?

File: src/getCoinCombination.test.js [hunk 3]
```diff
@@ -1,10 +1,53 @@
...
+  });
+
+  it('should return[0, 0, 2, 0]for 20cents(boundary for 10-cent coins)', () => {
+    expect(getCoinCombination(20)).toEqual([0, 0, 2, 0]);
+  });
 
-  it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+  it('should return[0, 0, 0, 1]for 25 cents(boundary for 25-cent coin)', () => {
```
Comment (djtyrf312) [inline src/getCoinCombination.test.js:50]: I would add 24 coins test also

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
small fix is required
